src/App.vue

```diff
@@ -1,25 +1,33 @@
 <template>
   <div
-    v-if="config"
     id="app"
+    v-if="config"
     :class="[
-      `theme-${config.theme}`,
-      `page-${currentPage}`,
-      isDark ? 'dark' : 'light',
+      'theme-' + config.theme,
+      'page-' + currentPage,
+      isDark ? 'is-dark' : 'is-light',
       !config.footer ? 'no-footer' : '',
     ]"
   >
-    <DynamicTheme v-if="config.colors" :themes="config.colors" />
+    <DynamicTheme :themes="config.colors" />
```

Comment:
This breaks everything when no colors is defined. Do you even tested it? 

![2025-02-02T14:25:11,746563437+01:00](https://github.com/user-attachments/assets/362b9c6d-adfa-49fa-8b72-a73554731dc6)